src/sqlfluff/utils/reflow/reindent.py

```diff
@@ -533,8 +533,10 @@ def _crawl_indent_points(
     cached_point: Optional[_IndentPoint] = None
     for idx, elem in enumerate(elements):
         if isinstance(elem, ReflowPoint):
+            following_class_types = elements[idx + 1].class_types
```

Comment:
Is there a possibility this will cause an `IndexError`?

Reply:
Not here - there's always a trailing `IndentBlock`. I'll add a comment to that effect...